src/ApiPort/CommandLine/ApiPortConfiguration.cs

```diff
@@ -80,11 +81,12 @@ private static string ArrayTransform(string arg, ConfigMapCount array)
         {
             foreach (var tag in array.Tags)
             {
-                var newKey = $"{array.ExpectedTag}:{array.Count++}";
+                var newKey = string.Format(CultureInfo.InvariantCulture, "{0}:{1}", array.ExpectedTag, array.Count++);
```

Comment:
I think we should switch to the invariant culture, but take a look at this: http://www.thomaslevesque.com/2015/02/24/customizing-string-interpolation-in-c-6/. Creating a helper method Invariant or something might be a bit cleaner. I hate losing the string interpolation :(